pyproject.toml

```diff
@@ -115,6 +114,10 @@ ignore = [
   "ANN102", # no type for cls
   "SIM108", # allow if-else in place of ternary
   "RET505", # allow else after return
+  "Q001", # bad quotes
+  "Q002", # bad quotes docstring
+  "Q003", # avoidable escaped quotes
```

Comment:
Are the ruff settings compatible with Black formatting we had previously?

Reply:
Yes for this project they are ok. I will add more of the conflicts for the later as well.

Reply:
why did these need to be added? in general, trying to keep a minimal ignore list is preferred

Reply:
We are adding their parent categories to be checked (like Q or E, W)